lib/qmp.ml

```diff
@@ -108,7 +112,7 @@ type message =
   | Command of (id option * command)
   | Error of (id option * error)
   | Success of (id option * result)
-  | Event of event
+  | Event of (event * event_data option)
```

Comment:
The new data field is at the same level of the existing timestamp and event fields, so I think the existing type event should be changed to support data as well:
``` 
type event = {
   timestamp: (int * int);
   event: string;
+  data: event_data option;
 }
```
and ` Event of event` should not be changed.

Reply:
Thanks, updated